### PR TITLE
Propose database client semconv stability working group

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

File: projects/database-client-semconv.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Deliverables
+
+* Mark the Database client semantic conventions as stable.
+* Update existing Database client instrumentation across all OpenTelemetry repositories to conform with the stable conventions.
+
+## Staffing / Help Wanted
+
+* @trask project lead
+* @lmolkova domain expert
```
Comment (trask) [inline projects/database-client-semconv.md:15]:
```suggestion
* @lmolkova domain expert
* TBD more domain experts
```

File: projects/database-client-semconv.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* TBD more domain experts
+* TBD technical committee sponsor
+* TBD engineers willing to prototype in at least two languages
+* TBD maintainers or approvers from languages committed to reviewing the prototypes
```
Comment (alanwest) [inline projects/database-client-semconv.md:19]: I volunteer as a maintainer willing to prototype, review, approve, etc.

**Review by lmolkova — APPROVED** [review]

File: projects/database-client-semconv.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+* Mark the Database client semantic conventions as stable.
+* Update existing Database client instrumentation across all OpenTelemetry repositories to conform with the stable conventions.
+
+## Staffing / Help Wanted
+
+* @trask project lead
+* @lmolkova domain expert
+* TBD more domain experts
+* TBD technical committee sponsor
```
Comment (jack-berg) [inline projects/database-client-semconv.md:17]: I volunteer as a TC sponsor.

I can also help prototype in java.

I'm committed to attend quite a few meetings already, so may not be able to make two meetings a week, but can commit to attending at least one per week.

Comment (sourabh1007) [inline projects/database-client-semconv.md:17]: I volunteer as a TC sponsor and maintainer.

Comment (trask) [reply projects/database-client-semconv.md:17]: hi @sourabh1007! you need to be an existing TC member or maintainer for that, but we'd love to have you join as a domain expert and/or prototyper

File: projects/database-client-semconv.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+## Deliverables
+
+* Mark the Database client semantic conventions as stable.
+* Update existing Database client instrumentation across all OpenTelemetry repositories to conform with the stable conventions.
+
+## Staffing / Help Wanted
+
+* @trask project lead
+* @lmolkova domain expert
+* TBD more domain experts
```
Comment (jcocchi) [inline projects/database-client-semconv.md:16]: I volunteer as a domain expert

**Comment by trask** [comment]
@lmolkova @jcocchi @sourabh1007 @alanwest @jack-berg based on the meeting poll, I've added the meetings to the OpenTelemetry public calendar for 6 weeks starting next Wed:

* Wed 9:30-10a Pacific Time
* Fri 8:30-9a Pacific Time

the public calendar has the Zoom link

the meeting notes doc also has the Zoom links: https://docs.google.com/document/d/1zTi_Z4WBisytPnXTpwtnGOK-4Gv5oSTuMHvIQ8--5hg

just ping me if you have any questions about logistics, and look forward to kicking this off next week!

**Review by svrnm — APPROVED** [review]

**Comment by dnaik-sfx** [comment]
Question for/about this working group: I'm looking at adding semantic conventions for some database query-level metrics, and was planning on adding the specs to the existing list of [Database Metrics](https://opentelemetry.io/docs/specs/semconv/database/database-metrics/) (within a new section). The language is a bit mixed with regard to whether general database metrics should be captured on that page, or if it should just be client metrics. Is the intention of this group to stabilize that list of database _client_ metrics, and keep that page isolated to those alone? Sorry if this is the wrong venue for this conversation; if there's a better place to talk, please let me know.

**Review by tedsuo — APPROVED** [review]

**Review by jack-berg — APPROVED** [review]

**Comment by trask** [comment]
> Question for/about this working group: I'm looking at adding semantic conventions for some database query-level metrics, and was planning on adding the specs to the existing list of [Database Metrics](https://opentelemetry.io/docs/specs/semconv/database/database-metrics/) (within a new section). The language is a bit mixed with regard to whether general database metrics should be captured on that page, or if it should just be client metrics. Is the intention of this group to stabilize that list of database _client_ metrics, and keep that page isolated to those alone? Sorry if this is the wrong venue for this conversation; if there's a better place to talk, please let me know.

hi @dnaik-sfx! can you open an issue in https://github.com/open-telemetry/semantic-conventions, and we can discuss specifics there?

**Pull request merged**